client/components/pages/HomePage/HomePage.js

```diff
@@ -22,8 +22,9 @@ export default function HomePage() {
       <Section>
         <Container>
           <Title size="1">
-            Home Page
+            Home page
```

Comment:
why this change?

Reply:
It was when is play with the project in the example.. Fixed